### DISCUSSION TITLE
Expose database functions through graphql

### DISCUSSION THREAD

**Discussion by fforres** [supabase/supabase#7243] [Questions]
Hey folks, haven't been able to find examples on this, I have been able to call functions through the `.rpc` methods in the JS SDK, but how would one go about exposing custom functions through the graphql API? 

![image](https://user-images.githubusercontent.com/952992/173087414-2c390924-1776-486f-979e-7a4a1b8fa2d6.png)

Is there anything way we have to do expose them via the graphql schema?

**Comment by lopezjurip** [comment]
Same question here, is it possible?

**Reply by willemclarke** [reply]
I second this, would love to know!!

**Reply by daanklijn** [reply]
@kiwicopple

**Reply by daanklijn** [reply]
The documentation from the screenshot above can be found on the [Edge Functions Best Practices](https://supabase.com/docs/guides/functions/best-practices) page.

**Reply by MCYouks** [reply]
Would love to keep updated on this!

**Comment by bh865** [comment]
Is this possible?
